tox.ini

```diff
@@ -1,12 +1,18 @@
 [tox]
-envlist = py27, py34, py35, py36, py37, flake8, black
+envlist = py{27,34,35,36,37}-mlflow{12,14,15,16,17}, flake8, black
 
 [testenv]
 sitepackages = False
 deps =
     pytest
     pytest-mock
     pytz
+    mlflow12: mlflow==1.2.0
+    mlflow13: mlflow>=1.3,<1.4
+    mlflow14: mlflow>=1.4,<1.5
+    mlflow15: mlflow>=1.5,<1.6
+    mlflow16: mlflow>=1.6,<1.7
+    mlflow17: mlflow>=1.7,<1.8
```

Comment:
It might be easier to read if we change all these to use the `~=` operator rather than having two limits. E.g., `mlflow12: mlflow~=1.2.0` (The last zero is important to indicate which range the `~` operator allows) is equivalent to `mlflow>=1.2,<1.3`.

Reply:
This makes sense. I think I cargo-culted the format above, but was wondering about the `~` way, thanks for the note!